### DISCUSSION TITLE
How can I restore a window from a minimized state to its normal display state？

### DISCUSSION THREAD

**Discussion by Xiaobaishushu25** [tauri-apps/tauri#7977] [Q&A] [answered]
And i also want to know how to listen window  winimization event？now i listen blur event and then judge it state to indirect judgment。
```
  await appWindow.listen('tauri://blur',async ()=>{
    if (await appWindow.isMinimized()){
      console.log(`Window is minimize,${await appWindow.isVisible()}`)
      //await appWindow.setMinimizable(false)
      // await appWindow.maximize()
      // await appWindow.setAlwaysOnTop(true)
      // await appWindow.setFocus()
    }
  })
```
I want to minimize this window and immediately restore normal display，thank you！

**Comment by FabianLars** [comment, marked as answer]
I _think_ you're asking for [`unminimize()`](https://tauri.app/v1/api/js/window/#unminimize)?
And yeah, there is no minimized event, blur is a good one, alternatively resize and move may also work (may depend on the platform)

**Reply by Xiaobaishushu25** [reply]
thanks！this function is what i want!
